tools/SetupFlow/DevHome.SetupFlow/Models/ConfigureTargetTask.cs

```diff
@@ -47,6 +47,8 @@ public class ConfigureTargetTask : ISetupTask
 
     private readonly AdaptiveCardRenderingService _adaptiveCardRenderingService;
 
+    private bool _isCompletionTelemetryLogged;

```

Comment:
Add a comment explaining what this boolean controls, and how the flag changes the telemetry behavior.

Reply:
sure I can do that